gui/src/components/WithLeftWorkspaceNav.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
+  import * as router from 'svelte-spa-router';
+
+  import Home from './mono/home.svelte';
+  import Layers from './mono/layers.svelte';
+  import Storage from './mono/database.svelte';
+  import Networking from './mono/networking.svelte';
+
+  export let workspaceId: string;
+  export let active: string;
+</script>
+
+<div>
+  <aside>
+    <button
```

Comment:
Can we create a `LinkButton` component or similar?

Reply:
Yeah, I'm gonna follow up with a code cleanup PR that turns these left navbar menus into reusable components including the buttons having their own component :)